packit.yaml

```diff
@@ -1,8 +1,9 @@
 upstream_project_url: https://github.com/cockpit-project/cockpit
 actions:
   post-upstream-clone:
-    # packit will overwrite the version in its "fix spec file" stage
-    - tools/create-spec --version 0 --build-all -o cockpit.spec tools/cockpit.spec.in
+    # HACK: packit's "fix spec file" stage does not kick in here, and it does not take the spec from the tarball
+    # https://github.com/packit/packit-service/issues/1515
+    - sh -exc 'tools/create-spec --version $(git describe) --build-all -o cockpit.spec tools/cockpit.spec.in'
```

Comment:
you're gonna want (need) `| tr - .` here

Reply:
Done.